modules/ycc_vm.py

```diff
@@ -365,6 +366,7 @@ def vm_argument_spec():
         disk_size=dict(type="int", required=False, default=10),
         secondary_disks_spec=dict(type="list", required=False),
         subnet_id=dict(type="str", required=False),
+        secondary_subnet_id=dict(type="str", required=False),
```

Comment:
Please add few words about option into [docs](https://github.com/arenadata/ansible-module-yandex-cloud/blob/aa9716486e717d293fdcd5f7423efa967e05b4b6/modules/ycc_vm.py#L154-L158) on top and in [readme](https://github.com/arenadata/ansible-module-yandex-cloud/blob/master/README.md#vm-managment). Also nice to mention there few words about public IP, DNS, security groups and FQDN for the second NIC.